terra2/assetlist.json

```diff
@@ -459,6 +459,34 @@
           "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/terra2/images/amproar.png"
         }
       ]
+    },
+    {
+      "description": "Local Money - the decentralized on/off ramp.",
+      "type_asset": "cw20",
+      "address": "terra1xzkel96e5e8vfmqw7valzdzzv9hqasfyslclvnmvxdejvpda3xwsskxzus",
+      "denom_units": [
+        {
+          "denom": "cw20:terra1xzkel96e5e8vfmqw7valzdzzv9hqasfyslclvnmvxdejvpda3xwsskxzus",
+          "exponent": 0
+        },
+        {
+          "denom": "whLOCAL",
+          "exponent": 6
+        }
+      ],
+      "base": "cw20:terra1xzkel96e5e8vfmqw7valzdzzv9hqasfyslclvnmvxdejvpda3xwsskxzus",
+      "name": "whLOCAL",
+      "display": "whLOCAL",
+      "symbol": "whLOCAL",
+      "logo_URIs": {
+        "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/terra2/images/local.png"
+      },
+      "coingecko_id": "local-money",
```

Comment:
Please remove this CoinGecko since it's being used on Kujira's LOCAL. The trace to Kujira (once defined herein) will allow interfaces tofind the coingecko id

Reply:
Will remove, what will link the whLOCAL on Terra2 to the native LOCAL on Kujira?

A bit of history of the LOCAL token, it was first minted as a cw20 on Terra before the UST collapse and the chain split. Then it was moved to Terra2 via Wormhole, a new version of the token with the same circulating supply was minted on Kujira, all the supply was transferred to a contract that allows the swap of whLOCAL to native LOCAL on Kujira, 1:1. whLOCAL can be transferred to Kujira via IBC/ICS20.

Reply:
Did all the LOCAL supply on Kujira come from the bridge and converted using the contract? Or is Kujira's LOCAL an additional mintage of LOCAL, where the 1:1 converting contract is just extra convenience for users?

Reply:
The linking is handled via the `traces` property, which points each asset's origin. For what it sounds like, LOCAL in Terra Classic was the first, so we'll treat this as the original--I see it is already defined in terra/assetlist.json.
Now Terra2's whLOCAL should have a `traces` property pointing it to Terra Classic (using type 'bridge'). For Kujira's LOCAL, I think it depends where the supply first came from.

Reply:
Yes, all tokens came from the bridge. We minted a new native token on Kujira with the same supply and sent it all to the contract where users can convert 1:1 from the wormhole version to the native on Kujira.

Reply:
In this case, I would say we should define the Terra2 to Kujira ibc-transferred $LOCAL, as well (with its trace origin pointing to Terra2, using type 'ibc'), and then this Kujira LOCAL that you already drafted would have a trace origin to the ibc Kujira LOCAL, using type 'wrapped'.
I hope that makes sense? If needed, I could try to define some of it for you